src/modules/renderScore.js

```diff
@@ -1,33 +1,44 @@
+import addApiScore from './addScore.js';
 import { name, score, leaderBoard } from './const.js';
+import getScore from './getApiScore.js';
 
 let scoreArr = [];
 const pushToLocal = () => {
   localStorage.setItem('leaderboard', JSON.stringify(scoreArr));
 };
 
 const addScore = () => {
-  const user = {};
-  user.name = name.value;
-  user.score = score.value;
-  scoreArr.push(user);
+  const users = {};
+  users.user = name.value;
+  users.score = score.value;
+  scoreArr.push(users);
+  addApiScore(users);
   pushToLocal();
 };
 
 const renderScore = () => {
   leaderBoard.innerHTML = '';
-  scoreArr.forEach((user) => {
-    const leader = `<li> ${user.name}: ${user.score}</li>`;
+  scoreArr.forEach((person) => {
+    const leader = `<li> ${person.user}: ${person.score}</li>`;
```

Comment:
**OPTIONAL** - Just like you did in the `refreshScore()` function below, instead of using the `scoreArr`, I suggest you call the function for fetching scores (since it is returning an array), and populate the UI using that array 👍 👍. Consider making your `renderScore()` method a method that takes in an array as input.